src/main/groovy/jenkins/automation/utils/CheckmarxSecurityJobBuilder.groovy

```diff
@@ -0,0 +1,63 @@
+package jenkins.automation.utils
+
+import javaposse.jobdsl.dsl.DslFactory
+import javaposse.jobdsl.dsl.Job
+
+
+class CheckmarxSecurityJobBuilder {
+
+    String name
+    String description
+    String baseUrl
+    String scanRepo
+    String checkmarxComment
+    Boolean vulnerabilityThresholdEnabled
+    String highThreshold
+    String mediumThreshold
+    String lowThreshold
+
+    Job build(DslFactory factory){
+        factory.job(name){
+            it.description this.description
+            
+            scm {
+                git {
```

Comment:
@marcesher I changed this to use BaseJobBuilder,, as far as multi-repo support, we normally scan one repo per Jenkins job and generate one Checkmarx report. I could probably add my single repo code to ScmUtils and make it reusable? 
@Lamine any comments on this?